src/domains/pay-wallet-app/api/npg-notifications/v1/_base_policy.xml.tpl

```diff
@@ -5,6 +5,16 @@
         <set-variable name="walletId" value="@(context.Request.MatchedParameters["walletId"])" />
         <set-variable name="npgNotificationRequestBody" value="@((JObject)context.Request.Body.As<JObject>(true))" />
         <set-variable name="orderIdBodyParam" value="@((string)((JObject)((JObject)context.Variables["npgNotificationRequestBody"])["operation"])["orderId"])" />
+        <validate-jwt query-parameter-name="sessionToken" failed-validation-httpcode="401" failed-validation-error-message="Unauthorized" require-expiration-time="true" require-signed-tokens="true" output-token-variable-name="jwtToken">
```

Comment:
_facciamo tutti gli scongiuri del caso :-)_
please report back to pay-wallet-app-v2 if we have to go back to the other apim

Reply:
This policy is not present in pay-wallet-app-v2 folder, will I have to porting all of this or we can leave as is?